eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java

```diff
@@ -383,6 +402,57 @@ private MutableBagMultimapFactory()
                 hashBagMultimap.put(key3, value3);
                 return hashBagMultimap;
             }
+
+            public <K, V> MutableBagMultimap<K, V> withAll(Multimap<K, V> multimap)
+            {
+                return new HashBagMultimap<K, V>(multimap);
+            }
+        }
+
+        public static final class MutableSortedBagMultimapFactory
+        {
+            private MutableSortedBagMultimapFactory()
+            {
+            }
+
+            public <K, V> MutableSortedBagMultimap<K, V> empty()
```

Comment:
Doesn't necessarily need to be in this PR, but this `empty()` method reminded me that for sorted structures, it's common to have an overload that takes a `Comparator`.